src/components/inputs/rich-text-input/rich-text-input.js

```diff
@@ -9,17 +9,22 @@ import plugins from '../../internals/rich-text-plugins';
 import html from '../../internals/rich-text-utils/html';
 import isEmpty from '../../internals/rich-text-utils/is-empty';
 
-const propsToState = props => ({
-  serializedValue: props.value || '',
-  internalSlateValue: html.deserialize(props.value || ''),
-});
-
 class RichTextInput extends React.PureComponent {
-  state = propsToState(this.props);
+  serializedValue;
+  internalSlateValue;
+
+  constructor(props) {
+    super(props);
+    this.serializedValue = props.value || '';
+    this.internalSlateValue = html.deserialize(this.props.value || '');
+  }
 
-  static getDerivedStateFromProps(props, state) {
-    if (props.value === state.serializedValue) return null;
-    return propsToState(props);
+  componentDidUpdate() {
+    if (this.props.value !== this.serializedValue) {
+      this.internalSlateValue = html.deserialize(this.props.value);
+      this.serializedValue = this.props.value;
+      this.forceUpdate();
```

Comment:
🤪🤪🤪🤪🤪🤪🤪🤪🤪🤪🤪🤪🤪🤪🤪🤪🤪🤪🤪

Reply:
Does this component still have a `shouldComponentUpdate` now?

Reply:
no, it's still a `PureComponent`

Reply:
Can you add a comment why we do forceUpdate? I think nobody will know in two months time.

Reply:
Sure 👍